presto-main/src/main/java/com/facebook/presto/operator/OrderByOperator.java

```diff
@@ -220,6 +220,12 @@ public Page getOutput()
         return page;
     }
 
+    @Override
+    public void close()
+    {
+        pageIndex.clear();
```

Comment:
Why not set to null (or both maybe)?

Reply:
It is final field. HashBuilderOperator does clear pageIndex similarly.